### ISSUE TITLE
java.lang.NullPointerException is displayed and Status Crash when the user slide the Fee bar to the maximum

### ISSUE THREAD

**Issue by ReneeAB** [status-im/status-react#2647]
### Description

*Summary*: 
java.lang.NullPointerException is displayed and Status Crash when the user slide the Fee bar to the maximum, I got two different error message each time (see screenshot) however, the issue is not that easy to reproduce.

*Type*: Bug

#### Expected behaviour
The  error message should not be displayed when the user changes the fee

#### Actual behavior
java.lang.NullPointerException is displayed when the user change the fee and try to send a transaction.

### Reproduction

- Open Status
- Select the "Chat" tab at the bottom of the screen.
- Select an existing chat on the list.
- Tap "/send" above the "type a message ..." field at the bottom of the page, the "Send transaction" screen is displayed.
- Slide the "Fee" bar right to left several times.
- Then, click the send button several times, the error message (see screenshot 1) appeared.
When I tried to reproduce the issue again I did the following:
- Leave the chat, the list of exiting chats are displayed.
- Select the same chat as previously
- Tap "/send", the "Send transaction" screen is displayed.
- Slide the "Fee"  to the maximum (faster), the error message 2) appeared.

Note: The issue was found on Android I haven't been able to reproduce on iOS.
1)
![screenshot_20171206-204624](https://user-images.githubusercontent.com/33925578/33685995-4ce67ebe-dacb-11e7-884a-35046ac806b2.png)

2)
![screenshot_20171206-205541](https://user-images.githubusercontent.com/33925578/33685972-377fc576-dacb-11e7-9094-6d7ba8b5ed6a.png)

3)
![screenshot_20171206-205528](https://user-images.githubusercontent.com/33925578/33686010-57fa7cba-dacb-11e7-8a46-022ca31200f4.png)

* Status version: 0.9.12 (1557.321)
* Operating System: Android, iOS

**Comment by annadanchenko** [comment]
@ReneeAB thanks for the bug report! Can you please rephrase this step and maybe split it into smaller steps: `Slide the "Fee" bar from right to left several time very fast the click on the blue arrow button to send several times (this is what I did the first time see screenshot 1) )`? I guess this is the key to reproduce the bug but I can't get it on my device, so would really appreciate if you can check for typos. Recording a video would be also very helpful (with "show touches" set to on). 

As another option, maybe you can install one of the nightly builds where we have extra logging enabled, so if app is using Wifi then video (1 picture per 1 second) and logcat will be sent to us. Just use this build http://artifacts.status.im:8081/artifactory/nightlies-local/im.status.ethereum-a68f06.apk and comment if issue can be reproduced. If yes then specify date-time-minutes when app was launched, country and device model so we can find your session. Thanks in advance!

**Comment by ReneeAB** [comment]
Hi @annadanchenko , I don't have the Android phone in my possession anymore, but I will try again later today on my phone (iPhone SE) and see if I can reproduce the bug there, if yes I will record it as you suggested. I updated the step above to make it easier to read I hope it helps.

**Comment by ReneeAB** [comment]
Hi @annadanchenko I want to try the second option and check if I can reproduce the ussue in the most recent version, what are the steps to install the most recent nightly build on Android and iOS? Do I keep the versions I installed from the TestFlight and  and Play Store?

**Comment by andytudhope** [comment]
Hi @ReneeAB. The latest nightly builds for android are here (just ctrl+f filter for December as they're not always in order): http://artifacts.status.im:8081/artifactory/nightlies-local/

We use diawi for iOS builds, but only core team members have access to it, so using the latest TestFlight version is the best you can do there.

**Comment by ReneeAB** [comment]
Hi, after installing the nightly build, I get a message that states that all interactions with the app is saved, so it means that I don't have to start anything manually right? Then where are the vidoes saved? I cannot find it on the phone.
Also I have seen that message again but this time I selected "/Send" and the error message was displayed before I had the time to do anything, unfortunatly I did not take a screenshot, will you guys be able to see it then?
I will try to reproduce the issue again before confirming, but it is really a tricky one.
